app/views/content/research.md

```diff
@@ -13,6 +13,14 @@ calls_to_action:
       file_type: PDF
       file_size: 258KB
       published_at: 18 November 2021
+  be-the-career-change:
+    name: attachment
+    arguments:
+      text: Read the Be the career change report summary
+      file_path: media/documents/BeTheCareerChangeCampaign_TeachingJan_Report_vFINAL.pdf 
```

Comment:
Pete has changed the filename (tidied it up a little) so the file-path will need amending